tests/system/model_monitoring/test_app.py

```diff
@@ -0,0 +1,150 @@
+        cls._kv_storage = ModelMonitoringWriter._get_v3io_client().kv
+        cls._tsdb_storage = ModelMonitoringWriter._get_v3io_frames_client(
+            cls._v3io_container
+        )
+
+    @classmethod
+    def custom_teardown_class(cls) -> None:
+        mlrun.mlconf.model_endpoint_monitoring.parquet_batching_max_events = (
+            cls._orig_parquet_batching_max_events
+        )
+
+    def _log_model(self) -> None:
+        dataset = load_iris()
+        train_set = pd.DataFrame(
+            dataset.data,
+            columns=dataset.feature_names,
+        )
+        self.project.log_model(
+            self.model_name,
+            model_dir=str((Path(__file__).parent / "assets").absolute()),
+            model_file="model.pkl",
+            training_set=train_set,
+        )
+
+    @classmethod
+    def _deploy_model_serving(cls) -> mlrun.runtimes.serving.ServingRuntime:
+        serving_fn = mlrun.import_function(
+            "hub://v2_model_server", project=cls.project_name, new_name="model-serving"
+        )
+        serving_fn.add_model(
+            cls.model_name,
+            model_path=f"store://models/{cls.project_name}/{cls.model_name}:latest",
+        )
+        serving_fn.set_tracking(
+            tracking_policy=TrackingPolicy(
+                default_batch_intervals=f"*/{cls.app_interval} * * * *",
+                application_batch=True,
+            ),
+        )
+        serving_fn.deploy()
+        return typing.cast(mlrun.runtimes.serving.ServingRuntime, serving_fn)
+
+    @classmethod
+    def _get_model_enpoint_id(cls) -> str:
+        endpoints = mlrun.get_run_db().list_model_endpoints(project=cls.project_name)
+        assert endpoints and len(endpoints) == 1
+        return endpoints[0].metadata.uid
+
+    @classmethod
+    def _generate_infer_input(cls) -> str:
+        return json.dumps({"inputs": [[0] * cls.num_features] * cls.max_events})
+
+    @classmethod
+    def _test_kv_record(cls, ep_id: str) -> None:
+        resp = ModelMonitoringWriter._get_v3io_client().kv.get(
+            container=cls._v3io_container, table_path=ep_id, key=cls.app_name
+        )
+        assert resp.output.item, "V3IO KV app data is empty"
+
+    @classmethod
+    def _test_tsdb_record(cls, ep_id: str) -> None:
+        df: pd.DataFrame = cls._tsdb_storage.read(
+            backend=_TSDB_BE,
+            table=_TSDB_TABLE,
+            start=f"now-{2 * cls.app_interval}m",
+        )
+        assert not df.empty, "No TSDB data"
+        assert (
+            df.iloc[0].endpoint_id == ep_id
+        ), "The endpoint ID is different than expected"
+
+    @classmethod
+    def _test_v3io_records(cls, ep_id: str) -> None:
+        cls._test_kv_record(ep_id)
+        cls._test_tsdb_record(ep_id)
+
+    def test_app_flow(self) -> None:
+        self.project = typing.cast(mlrun.projects.MlrunProject, self.project)
+        self.project.set_model_monitoring_application(
```

Comment:
Please check out this branch:
https://github.com/jond01/mlrun/tree/tests/app-sys-test-patch

Specifically, see the last commit:
https://github.com/jond01/mlrun/commit/8eb79baafb5f1cdfd82768fb13cc795649146b5c

It sets the image tag: `mlrun/mlrun:1.6.0-rc1`. I believe it will solve the issue, but it shouldn't be committed.